SETUP.md

```diff
@@ -1 +1,36 @@
+# Getting setup
 
+We will be using Anaconda to install and manage a Python 'environment' for us to work in. Rather than using any pre-installed Python 
+interpretors this has the advantage that it will not conflict with any existing libraries, and everyone will be using the same verisons.
+
+## 1. Install Anaconda
+
+If you haven't already got Anaconda installed, download and install it from: https://www.continuum.io/downloads. Choose Python 3.5 as this 
```

Comment:
I guess you will lose 'Anaconda prompt' on windows perhaps? I can't remember and I don't have a windows machine handy.